controllers/defaults/resources.go

```diff
@@ -66,18 +66,32 @@ var (
 		},
 		"ocs-metrics-exporter": {
 			Requests: corev1.ResourceList{
-				"memory": resource.MustParse("50Mi"),
-				"cpu":    resource.MustParse("50m"),
+				"memory": resource.MustParse("512Mi"),
+				"cpu":    resource.MustParse("250m"),
+			},
+			Limits: corev1.ResourceList{
+				"memory": resource.MustParse("1.5Gi"),
```

Comment:
Why does the exporter go till 1.5Gi, seems a lot.

Reply:
We have seen memory consumption reaching till `1.25Gi` for Scale tests ran during testing, therefore setting a safe limit of `1.5Gi`

Reply:
Need a bit more investigation, I understand even w/o this PR the exporter has no bounds but we better take this moment to re-look at the exporter and come to a conclusion for bumping the values.

Reply:
I would suggest to take this PR in and work on parallel to identify any memory leaks or such issues that could lead to a memory spike in exporter.